.github/workflows/android_build.yml

```diff
@@ -0,0 +1,65 @@
+name: Android Build
+
+on: 
+  push:
+    branches: [ 'master' ]
+  
+  pull_request:
+    branches: [ 'master' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: gradle
+    - name: Execute permission for gradlew
+      run: chmod +x gradlew
+      
+    - name: Build 
+      run: ./gradlew build
+      
+    - name: Run Unit Test
+      run: ./gardlew test  --stacktrace
```

Comment:
```suggestion
      run: ./gradlew test  --stacktrace
```

Reply:
Thank for pointing that out. I should've been more careful.